class_exercise.sh

```diff
@@ -0,0 +1,32 @@
+# !/bin/bash
+
+# Christina La ckla@usfca.edu
+# September 12, 2017
+
+# Setting up directories
+
+echo "Setting up directories"
+
+mkdir code data data/raw_data output output/figures output/tables
+
+echo "Done making directories"
+
+cd data/raw_data/
+
+echo "Now in raw_data"
+
+curl -L http://npk.io/aMPCOe+ -o filename.fasta
```

Comment:
The idea was to replace `filename.fasta` with a better name.

Reply:
You can also download directly using syntax like: `curl -L http://npk.io/aMPCOe+ -o data/raw_data/filename.fasta`